client/views/6-AddNewContactView.js

```diff
@@ -0,0 +1,365 @@
+import React, { Component } from 'react';
```

Comment:
Don't number the files, even if they have the navigation order like noted it may change and then it becomes not mantainable.